MainPage/MainPage.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap" rel="stylesheet">    
+    <script src="https://kit.fontawesome.com/4c4a173085.js" crossorigin="anonymous"></script>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="MainPage.css">
+    <title>Products</title>
+</head>
+<body>
+    <main class="container">
+        <header>
+            <h3 class="head">Welcome</h3>
+            <div class="search-box">
+                <input type='text' class="search-txt"></input>               
+                <i class="fas fa-search searchIcon"></i>
+            </div>
+        </header> 
+        
+        <nav class="navbar">
+            <ul class="list">
+                <li class="allProduct active" id="allProduct">All</li>
+                <li class="makeupBtn" id="makeupBtn">MakeUp</li>
+                <li class="jewelryBtn" id="jewelryBtn">Jewelry</li>
+            </ul>
+        </nav>
+
+        <section class="slider"></section>
+        <section class="recom"></section>
+        <footer>
+            <div class="icons">
+                <a href="../Index.html"></a><i class="fas fa-home"></i>
+                <a href="../ShoppingCart/ShoppingCart.html"></a><i class="fas fa-shopping-cart"></i>
```

Comment:
Just put the icon inside the <a>icon</a> :) it will solve the problem
                <a href="../Index.html"><i class="fas fa-home"></i></a>
                <a href="../ShoppingCart/ShoppingCart.html"><i class="fas fa-shopping-cart"></i></a>

Reply:
right, I will solve that